docs/.vitepress/theme/style.css

```diff
@@ -8,8 +8,17 @@
  */
 
 @font-face {
-  font-family: "Geist";
-  src: url("/assets/GeistVariableVF.woff2") format("woff2");
```

Comment:
Unused (I tested this but wasn’t a fan). Forgot to clean up this style, but fortunately unused `@font-face` tags are just ignored